### DISCUSSION TITLE
When are you going to get unbanned on Scratch?

### DISCUSSION THREAD

**Discussion by Chiroyce1** [CubeyTheCube/scratchclient#42] [Q&A]
# 🤔

**Comment by Quantum-Codes** [comment]
He is banned?

**Reply by Chiroyce1** [reply]
Yeah... why else would he be inactive?

**Reply by Quantum-Codes** [reply]
Oh ok

**Reply by pikachub2005** [reply]
> Yeah... why else would he be inactive?

I doubt you guys remember me but I don't think I'm ever getting unbanned ;-;

**Reply by Quantum-Codes** [reply]
I do remember you and its confirmed I never being unbanned
However this isnt the place to talk these things
